spec/lib/stripe/credit_card_remover_spec.rb

```diff
@@ -89,13 +89,16 @@
     end
 
     context 'Stripe customer does not exist' do
-      it 'deletes the credit card clone' do
-        allow(Stripe::Customer).to receive(:retrieve).and_return(nil)
+      let(:non_existing_customer_id) { 'non_existing_customer_id' }
 
-        expect_any_instance_of(Stripe::CreditCardCloneDestroyer).to receive(:destroy_clones).with(
-          credit_card
-        )
-        Stripe::CreditCardRemover.new(credit_card).call
+      before do
+        credit_card.update_attribute :gateway_customer_profile_id, non_existing_customer_id
+      end
+
+      it 'deletes the credit card clone' do
```

Comment:
`±` I think this description was wrong ;)
```suggestion
      it 'raises an error' do
```

Reply:
Agree - at least it makes more sense to me, that it triggers an error...

Reply:
More to the point: we aren't testing that the card was deleted (presumably it doesn't get deleted in this case)